src/pyTRLCConverter/rst_converter.py

```diff
@@ -0,0 +1,701 @@
+from pyTRLCConverter.log_verbose import log_verbose
+
+# Variables ********************************************************************
+
+# Classes **********************************************************************
+
+class RstConverter(BaseConverter):
+    """
+    RstConverter provides functionality for converting to a reStructuredText format.
+    """
+    OUTPUT_FILE_NAME_DEFAULT = "output.rst"
+    TOP_LEVEL_DEFAULT = "Specification"
+
+    def __init__(self, args: any) -> None:
+        # lobster-trace: SwRequirements.sw_req_rst
+        """
+        Initializes the converter.
+
+        Args:
+            args (any): The parsed program arguments.
+        """
+        super().__init__(args)
+
+        # The path to the given output folder.
+        self._out_path = args.out
+
+        # The file descriptor for the output file. Used in single and multiple document mode.
+        self._fd = None
+
+        # The base level for the headings. Its the minimum level for the headings which depends
+        # on the single/multiple document mode.
+        self._base_level = 1
+
+        # For proper reStructuredText formatting, the first written part shall not have an empty line before.
+        # But all following parts (heading, table, paragraph, image, etc.) shall have an empty line before.
+        # And at the document bottom, there shall be just one empty line.
+        self._empty_line_required = False
+
+    @staticmethod
+    def get_subcommand() -> str:
+        # lobster-trace: SwRequirements.sw_req_rst
+        """
+        Return subcommand token for this converter.
+
+        Returns:
+            str: Parser subcommand token
+        """
+        return "rst"
+
+    @staticmethod
+    def get_description() -> str:
+        # lobster-trace: SwRequirements.sw_req_rst
+        """
+        Return converter description.
+
+        Returns:
+            str: Converter description
+        """
+        return "Convert into reStructuredText format."
+
+    @classmethod
+    def register(cls, args_parser: any) -> None:
+        # lobster-trace: SwRequirements.sw_req_rst_multiple_doc_mode
+        # lobster-trace: SwRequirements.sw_req_rst_single_doc_mode
+        # lobster-trace: SwRequirements.sw_req_rst_sd_top_level_default
+        # lobster-trace: SwRequirements.sw_req_rst_sd_top_level_custom
+        # lobster-trace: SwRequirements.sw_req_rst_out_file_name_default
+        # lobster-trace: SwRequirements.sw_req_rst_out_file_name_custom
+        """
+        Register converter specific argument parser.
+
+        Args:
+            args_parser (any): Argument parser
+        """
+        super().register(args_parser)
+
+        BaseConverter._parser.add_argument(
+            "-e",
+            "--empty",
+            type=str,
+            default=BaseConverter.EMPTY_ATTRIBUTE_DEFAULT,
+            required=False,
+            help="Every attribute value which is empty will output the string." \
```

Comment:
Shouldn't be a blank at the end instead of a "." ? 

The default "(.."  f-string is appended to it,